### ISSUE TITLE
RandomPages doesn't change

### ISSUE THREAD

**Issue by epitron** [WardCunningham/remodeling#15]
http://wiki.c2.com/?RandomPages

Is this not implemented yet? Or is it a caching issue or something?

**Comment by WardCunningham** [comment]
A reimplementation is called for in #2.

The original ran as a cron job that applied some heuristic exploiting access to the database. We stopped running that job and don't intend to resume it but would consider new heuristics once we make more global state available in the client code.

**Comment by WardCunningham** [comment]
We decided not to add more global state to duplicate old heuristics. Instead 9f5a96c46 uses the existing names.txt to choose random pages immediately and indiscriminately.

**Comment by epitron** [comment]
Awesome! I was actually going to suggest using names.txt, but the issue was closed.

**Comment by WardCunningham** [comment]
When I wrote the first version of RandomPages about half of the results would be small home pages. I engineered a revised version that omitted home pages entirely. Apparently in the years that followed idea pages grew much faster than home pages because the mix on any given random selection now favor ideas.

![image](https://cloud.githubusercontent.com/assets/12127/20248962/83f22d8a-a9a4-11e6-8e35-445810e5ded7.png)
